internal/pkg/log/logger.go

```diff
@@ -23,16 +24,18 @@ type zapLogger struct {
 	component     string
 }
 
-func loggerFromZapCore(core zapcore.Core, attrs ...attribute.KeyValue) *zapLogger {
-	var fields []zap.Field
-	ctxattr.AttrsToZapFields(attrs, &fields)
-	core = core.With(fields)
+func loggerFromZapCore(core zapcore.Core) *zapLogger {
```

Comment:
The `loggerFromZapCore` had 2 responsibilities:
- Create logger.
- Setup fields - it was used only once - moved to the `With` method.